### ISSUE TITLE
Add the new TWRP App to f-droid.org

### ISSUE THREAD

**Issue by ekaris** [TeamWin/twrpme#24]
"F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform. The client makes it easy to browse, install, and keep track of updates on your device."

https://f-droid.org

**Comment by kannayuki** [comment]
I don't think that the app is open source, @ekaris. Which is actually a bit weird, since TWRP is an open-source project to begin with.

**Comment by herbsmn** [comment]
That's sad

**Comment by lucasfcnunes** [comment]
I am missing that too...
Hope they look at that. I'm trying to keep Google Services out of some devices... they do have an *.apk so that's not a real issue. Nevertheless, it would be nice to have it on [](https://f-droid.org)

**Comment by frankholguin** [comment]
How do i flash f-droid as system app?? I have f-droid as an app but still have play store.. Also would like to know step by step how to root my phone? Thank you..

LG K51 Android version 10 kernel 4.9.190+
